src/get_ensemble_output.R

```diff
@@ -3,41 +3,69 @@ library(hubVis)
 library(hubEnsembles)
 library(hubData)
 library(dplyr)
+library(purrr)
+library(here)
 
-
-hub_path <- "/Users/dk29776/Dropbox/UTAustin/flu-metrocast"
+hub_path <- here("..", "flu-metrocast")
 hub_con <- connect_hub(hub_path)
 
-model_names <- hub_con %>%  
-  select(model_id) %>% 
-  distinct() %>% 
+model_names <- hub_con %>%
+  select(model_id) %>%
+  distinct() %>%
   collect()
 
 model_names
 
-reference_date = "2025-03-01"
+## specify the reference_date to generate the ensemble model for
+reference_date = "2025-04-12"
```

Comment:
I couldn't get this date to work in my version of the repo, but once I changed it, the script worked for me.